mandatory/4-tax.js

```diff
@@ -4,8 +4,14 @@
   A business requires a program that calculates how much the price of a product is including sales tax
   Sales tax is 20% of the price of the product.
 */
+/*
+1.2*price of priceOfProduct(price product INC SALES TAX)
+*/
+
+function calculateSalesTax(salesTax, priceOfProduct) {
 
-function calculateSalesTax() {}
+  return 0.2 * priceOfProduct;
```

Comment:
```suggestion
  return 1.2 * priceOfProduct;
```
This is the correct logic you have in your comment above